src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1593,6 +1593,17 @@ static Library()
                     checkRuntimeValues: DeferRuntimeValueChecking,
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: Year)
+            },
+            {
+                BuiltinFunctionsCore.Language,
+                StandardErrorHandling<FormulaValue>(
+                    BuiltinFunctionsCore.Language.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: DeferRuntimeTypeChecking,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnEmptyStringIfAnyArgIsBlank,
```

Comment:
>ReturnEmptyStringIfAnyArgIsBlank [](http://example.com/codeflow?start=51&length=32)

Language function doesn't take any argument. Is this still the right option?

Reply:
You can use NoErrorHandling. Take a look at Pi, which has a similar signature.

Reply:
updated this. thanks